pykerb/kerberos.c

```diff
@@ -31,6 +31,22 @@ PyObject *BasicAuthException_class;
 PyObject *PwdChangeException_class;
```

Comment:
I would be good to `#include` py3c.h header (after `#include <Python.h>`) to make maintaining py2/py3 compatibility easier.